### PR TITLE
Add JwtSecret to JsonRpcClient

### PR DESCRIPTION
**Pull request by smartprogrammer93**
Add JwtSecret to JsonRpcClient in the tests

### CODE REVIEW

File: src/Nethermind/Nethermind.Overseer.Test/JsonRpc/JsonRpcClient.cs [hunk 1]
```diff
@@ -76,6 +104,15 @@ public async Task<JsonRpcResponse<T>> PostAsync<T>(string method, object[] @para
...
 
+        private static byte[] DecodeSecret(string hexSecret)
+        {
+            int start = hexSecret.StartsWith("0x") ? 2 : 0;
+            return Enumerable.Range(start, hexSecret.Length - start)
+                .Where(x => x % 2 == 0)
+                .Select(x => Convert.ToByte(hexSecret.Substring(x, 2), 16))
+                .ToArray();
+        }
+
```
Comment (LukaszRozmej) [inline src/Nethermind/Nethermind.Overseer.Test/JsonRpc/JsonRpcClient.cs:115]: `Bytes.FromHexString` ?

Comment (smartprogrammer93) [reply src/Nethermind/Nethermind.Overseer.Test/JsonRpc/JsonRpcClient.cs:115]: just copy pasted from here
https://github.com/NethermindEth/nethermind/blob/d1bed2f39891e41bba8a79acaf1872d505112644/src/Nethermind/Nethermind.Core/Authentication/MicrosoftJwtAuthentication.cs

Comment (smartprogrammer93) [reply src/Nethermind/Nethermind.Overseer.Test/JsonRpc/JsonRpcClient.cs]: Applied suggestion

**Review by MarekM25 — CHANGES_REQUESTED** [review]
The goal of this test was supposed to work without JWT on non-auth port. I don't want to modify JsonRpcClient just because of that. I think we can close this PR

**Comment by smartprogrammer93** [comment]
Closing per request

**Pull request closed without merging**